decision_making.md

```diff
@@ -0,0 +1,41 @@
+
+## Formal aspects of decision making
+
+The decision making process is intended to balance broad participation of stakeholders with agility. While the size of decision making bodies may vary (e.g. software projects can range from a handful to dozens of decision makers), all need to find this balance.
+
+All official Jupyter projects, working groups, governing bodies, and official initiatives must have a clearly defined, and documented decision making body. Formally, these decision making bodies make decisions using consensus seeking with an option to call a vote to move the decision forward.
+
+Depending on the governing body, decisions may be proposed by members of the decision making body, or by a broader group of stakeholders/contributors.
+
+Decision making, even when it begins with simple informal discussion, starts with consensus seeking. The goal of this discussion phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus seeking process is predicated on all stakeholders having their voice heard; teams must be proactive in providing opportunity for all relevant stakeholders to provide input. If the decision making body arrives at informal consensus, they may immediately move to document and enact the decision. This is the consensus seeking phase.
+
+At any time during the consensus seeking phase, any member of the decision making body can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form. From that point, members have a minimum of 7 days to vote. The governing body may consider longer voting periods as necessary for special circumstances, or shorter periods if all voting members are present. The result will be determined by a simple majority for binary decisions (with no quorum) and ranked choice for multiclass decisions (one among many, or many among many). The sponsor may update the proposal at any point during the voting period, in which case, the voting period will be reset.
+
+The various Jupyter decision making bodies often have vast scopes and members are often busy. As a result, there are typically a subset of members who have the time, interest, and expertise to dive into the details of a given decision. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously on a per-decision basis, in a manner that supports our goals of agility and participation. We have found that formal quorums slow decision making substantially, and don’t actually guarantee true participation.
```

Comment:
"No quorum" is not a healthy governance practice unless you are coupling it with a specific time frame (7 days) with formal notification of stakeholders that a vote is taking place.

I would completely strike the last sentence as the statements are anecdotal. Not meeting quorums are a red flag that a decision making body is ineffective. Quorums don't guarantee participation though without them the likelihood of a minority railroading through a decision is much higher. Instead of no quorum, it would be more responsible to set a minimum quota of 30%. [For reference from BoardSource](https://boardsource.org/resources/board-meeting-quorum/)

Reply:
Also applying a "no quorum" to a Board of Directors falls short of responsible governance. For example, 1 board member could make decisions on behalf of the entire board. Some level of quorum guards against this possibility. With no quorum, someone with a conflict of interest could make an self-interested (even if unintentional) individual decision that could jeopardize the project.

Reply:
Furthermore, having some form a quorum requirement is a protection for the board members against being pressured or intimidated for action, even if they would rather not take individual action. Once there is no quorum requirement, an individual can be pressured to take action where they would otherwise want to seek board discussion.

Reply:
Here's a gist that calculates quorum and simple majority on decisions for a group size and quorum level: https://gist.github.com/willingc/dc56d5d11b741e78859e589a59523821 

It's unclear to me what is driving the need for "no quorum". Minimal quorum isn't a heavy burden.